### DISCUSSION TITLE
Why do I get a "bad peer" error while downloading index.db.bz2?

### DISCUSSION THREAD

**Discussion by rontarrant** [MidnightBSD/src#227] [Q&A]
In the second stage of the installation (in VirtualBox) I got this error:
ntpd[776]: 1 out of 4 peers valid
ntpd[776]: bad peer 0.midnightbsd.pool.ntp.org (142.126.225.61)
ntpd[776]: bad peer 1.midnightbsd.pool.ntp.org (162.159.200.1)
ntpd[776]: bad peer 2.midnightbsd.pool.ntp.org (23.133.168.246)

And then the install stops. Anyone know what's going on with this?

**Comment by laffer1** [comment]
That isn't the real issue if it's stopping on boot.  The ntpd daemon can fail to acquire time servers and the OS will still work.  The root cause of that error is that the pool is random ntpd servers.  If someone reboots their server or takes one down, it can take time to remove from the pool.  

I would need to know what else is on the screen.  Did it start asking questions in the first boot script to setup a graphical desktop, etc?
